### PR TITLE
A lot of changes to Publications, Workshop, Media Coverage, Program, …

### PR DESCRIPTION
**Pull request by krishnapandeykp**
…Testimonials, Infrastructure

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -30,6 +30,7 @@ import { OutreachComponent } from './views/outreach/outreach.component';
 import { StudentsSeminarComponent } from './views/students-seminar/students-seminar.component';
 import { MediaCoverageComponent } from './views/media-coverage/media-coverage.component';
 
+
```
Comment (architbubber) [inline src/app/app.module.ts]: Remove this empty line

Comment (krishnapandeykp) [reply src/app/app.module.ts]: Done

File: src/app/views/activities-and-events/activities-and-events.component.html [hunk 2]
```diff
@@ -1,101 +1,40 @@
...
-    <a href="">Read more..</a>
-  </div>
-  <div class="card align-self-center p-5 m-1 shadow my-2 ">
-    s in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia, looked up one of the more obscure Latin words, consectetur, from a Lorem Ipsum passage, and going through the cites of the word in classical literature, discovered the undoubtable source. Lorem Ipsum comes from sections 1.10.32 and 1.10.33 of "de Finibus Bonorum et Malorum" (The Extremes of Good and Evil) by Cicero, written in 45 BC. This book is a treatise on the theory of ethics, very popular during the Renaissance. The first line of Lorem Ipsum, "Lorem ipsum dolor sit amet..", comes from a line in section 1.1<br>~ Archit
-    <a href="">Read more..</a>
-  </div>
-</div> -->
-
+<button class="scroll-to-top" #scrollToTopButton (click)="scrollToTop()">↑</button>
```
Comment (architbubber) [inline src/app/views/activities-and-events/activities-and-events.component.html]: We do not need this #scrollToTopButton. 
Just add in class "d-none d-md-block"

File: src/app/views/activities-and-events/activities-and-events.component.ts [hunk 3]
```diff
@@ -1,11 +1,36 @@
...
   selector: 'app-activities-and-events',
   templateUrl: './activities-and-events.component.html',
   styleUrls: ['./activities-and-events.component.scss']
 })
-export class ActivitiesAndEventsComponent {
+export class ActivitiesAndEventsComponent implements OnInit {
+  years: string[] = [];
+  selectedYear: string = '';
+  eventData: any;
+  @ViewChild('scrollToTopButton') scrollToTopButton?: ElementRef;
```
Comment (architbubber) [inline src/app/views/activities-and-events/activities-and-events.component.ts:13]: Not needed

File: src/app/views/homepage/homepage.component.html [hunk 4]
```diff
@@ -49,14 +49,14 @@ <h1 class="card-header">
...
-        <li><a class="nav-link" routerLink="/events/workshop-and-meetings">Seminars</a></li>
+        <li><a class="nav-link" routerLink="/events/monsoon-school" href="#" onclick="window.scrollTo(0, 0)">Monsoon Schools</a></li>
+        <li><a class="nav-link" routerLink="/events/workshop-and-meetings" href="#" onclick="window.scrollTo(0, 0)">Workshops</a></li>
+        <li><a class="nav-link" routerLink="/events/workshop-and-meetings" href="#" onclick="window.scrollTo(0, 0)">Seminars</a></li>
       </ul>
     </span>
     <div class="vr d-md-block d-none p-0"></div>
     <div class="col-3 d-md-block d-none offset-md-1 justify-content-center align-items-center" >
-      Opportunities don't happen, you create them.
+      "Opportunities are like sunrises. If you wait too long, you miss them."
```
Comment (architbubber) [inline src/app/views/homepage/homepage.component.html]: Did you check  the output on ui ?
Remove the quotes.
Did you check the new statement is lengthy hence it is looking bulging on smaller screens ?

File: src/app/views/media-coverage/media-coverage.component.ts [hunk 5]
```diff
@@ -8,4 +8,26 @@ import { media_coverage } from '../../../data-entries/json/media-coverage';
 })
 export class MediaCoverageComponent {
   media_coverage = media_coverage;
+
+  @ViewChild('scrollToTopButton') scrollToTopButton!: ElementRef;
```
Comment (architbubber) [inline src/app/views/media-coverage/media-coverage.component.ts:12]: Remove this and related

File: src/app/views/media-coverage/media-coverage.component.html [hunk 6]
```diff
@@ -1,15 +1,23 @@
...
+      <h5 class="text">{{ item.Details }}</h5>
+      <div class="col-12 d-flex">
+        <a [href]="item.link[0]" target="_blank" class="ms-auto">Click to view full article</a>
       </div>
-    </ng-container>
-  </div>
+    </div>
+  </ng-container>
 </div>
+<button class="scroll-to-top" #scrollToTopButton (click)="scrollToTop()">↑</button>
```
Comment (architbubber) [inline src/app/views/media-coverage/media-coverage.component.html]: #scrollToTopButton  ??

File: src/app/views/media-coverage/media-coverage.component.scss [hunk 7]
```diff
@@ -1,52 +1,23 @@
...
-  html{
-    -ms-overflow-style: none;  /* IE and Edge */
-    scrollbar-width: none;  /* Firefox */
-  }
-  ::-webkit-scrollbar {
-    display: none;/* Hide scrollbar for Chrome, Safari and Opera */
-    width: 0px;
-    background: transparent; /* make scrollbar transparent */
-  }
+.text{
```
Comment (architbubber) [inline src/app/views/media-coverage/media-coverage.component.scss:21]: rename class to text-justify

File: src/app/views/outreach/outreach.component.html [hunk 8]
```diff
@@ -1,14 +1,19 @@
...
+    <div class="row shadow border card border-1 mb-2 col-12 col-md-10 border-dark p-3 ">
+      <h4 class="text"><b>{{ item.title }}</b></h4>
+      <h5 class="text">{{ item.details }}</h5>
+      <div class="col-12 d-flex">
+        <a [href]="item.link" target="_blank" class="ms-auto">Click here to view</a>
+      </div>
+    </div>
+  </ng-container>
+</div>
+<button class="scroll-to-top" #scrollToTopButton (click)="scrollToTop()">↑</button>
```
Comment (architbubber) [inline src/app/views/outreach/outreach.component.html]: #scrollToTopButton

File: src/app/views/outreach/outreach.component.scss [hunk 9]
```diff
@@ -1,6 +1,23 @@
...
+  height: 40px;
+  font-size: 20px;
+  cursor: pointer;
+  outline: none;
+  transition: background-color 0.3s ease;
+}
+.scroll-to-top:hover {
+  background-color: #004080;
+}
+.text{
```
Comment (architbubber) [inline src/app/views/outreach/outreach.component.scss]: ?

File: src/app/views/testimonials/testimonials.component.scss [hunk 10]
```diff
@@ -1,116 +1,29 @@
...
-    padding: 40px;
-    position: relative;
 
-    &:hover:before {
-      width: 100%;
-    }
+.title h1 {
+  text-transform: uppercase;
+}
+.card .row g-0 .col-md-5.col-sm-12 img {
```
Comment (architbubber) [inline src/app/views/testimonials/testimonials.component.scss]: g-0 ?? shouldn't this be '.g-0'

File: src/app/views/testimonials/testimonials.component.scss [hunk 11]
```diff
@@ -1,116 +1,29 @@
...
-    bottom: 0px;
-    height: 2px;
-    width: 0%;
-    background: #333333;
-    right: 0px;
-    -webkit-transition: width 0.4s;
-    transition: width 0.4s;
-    }
-  }
+.card .row g-0 .col-md-7.col-sm-12 .card-body h5.card-title {
```
Comment (architbubber) [inline src/app/views/testimonials/testimonials.component.scss]: ?

File: src/app/views/testimonials/testimonials.component.scss [hunk 12]
```diff
@@ -1,116 +1,29 @@
...
-    p{
-      font-size: 12px;
-    }
-    a{
-      // font-family: 'Montserrat', sans-serif;
-      font-size: 12px;
-      text-transform: uppercase;
-      color: #666666;
-      margin-top: 10px;
+.card .row g-0 .col-md-7.col-sm-12 .card-body p.card-text {
```
Comment (architbubber) [inline src/app/views/testimonials/testimonials.component.scss]: ?

File: src/app/views/activities-and-events/activities-and-events.component.html (hunk 2 above)
Comment (krishnapandeykp) [reply src/app/views/activities-and-events/activities-and-events.component.html]: Done. Everywhere.

File: src/app/views/homepage/homepage.component.html (hunk 4 above)
Comment (krishnapandeykp) [reply src/app/views/homepage/homepage.component.html:59]: Yup I did the check, but that was asked so I did.

File: src/app/views/activities-and-events/activities-and-events.component.ts (hunk 3 above)
Comment (krishnapandeykp) [reply src/app/views/activities-and-events/activities-and-events.component.ts:13]: Removed

File: src/app/views/homepage/homepage.component.html (hunk 4 above)
Comment (krishnapandeykp) [reply src/app/views/homepage/homepage.component.html]: Indeed. But it was suggested to put a lengthy one...so I did. but now reverted back.

File: src/app/views/media-coverage/media-coverage.component.html (hunk 6 above)
Comment (krishnapandeykp) [reply src/app/views/media-coverage/media-coverage.component.html]: Done

File: src/app/views/media-coverage/media-coverage.component.scss (hunk 7 above)
Comment (krishnapandeykp) [reply src/app/views/media-coverage/media-coverage.component.scss:21]: Not Working...so did it. Now also I have checked but it's not working.

File: src/app/views/outreach/outreach.component.html (hunk 8 above)
Comment (krishnapandeykp) [reply src/app/views/outreach/outreach.component.html]: Done.

File: src/app/views/outreach/outreach.component.scss (hunk 9 above)
Comment (krishnapandeykp) [reply src/app/views/outreach/outreach.component.scss]: It's the same taken from Media Coverage, Now I have removed it. Probably will not be needed in this page for now.

File: src/app/views/testimonials/testimonials.component.scss (hunk 10 above)
Comment (krishnapandeykp) [reply src/app/views/testimonials/testimonials.component.scss]: Yup..fixed.

File: src/app/views/testimonials/testimonials.component.scss (hunk 11 above)
Comment (krishnapandeykp) [reply src/app/views/testimonials/testimonials.component.scss]: Fixed.

File: src/app/views/testimonials/testimonials.component.scss (hunk 12 above)
Comment (krishnapandeykp) [reply src/app/views/testimonials/testimonials.component.scss]: Fixed.

File: src/app/views/activities-and-events/activities-and-events.component.ts (hunk 3 above)
Comment (krishnapandeykp) [reply src/app/views/activities-and-events/activities-and-events.component.ts:13]: Removing it, not helping the cause. May have to implement in other way altogether.

File: src/app/views/media-coverage/media-coverage.component.ts (hunk 5 above)
Comment (krishnapandeykp) [reply src/app/views/media-coverage/media-coverage.component.ts:12]: May have to implement in other way altogether.

**Review by ICWaR — APPROVED** [review]

**Pull request merged**